resources/lib/SUBUtilities.py

```diff
@@ -69,6 +69,34 @@ def clean_title(item):
     item["title"] = re.sub(r'\([^\)]+\)\W*$', '', item["title"])
     item["tvshow"] = re.sub(r'\([^\)]+\)\W*$', '', item["tvshow"])
 
+def takeTitleFromFocusedItem():
+    try:
```

Comment:
Basically you are right, there is no need for this actually.
I put it just to be sure if there will be kind of exception for some reason with one of the getInfoLable functions.